.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: node_js
 node_js:
   - '8'
-  - '7'
-  - '6'
```

Comment:
Are we locking `sox` to node 8 just for boom?

Reply:
I double-checked our repos, and we don't use `sox` in anything less than node 8, so I figured it was safe.